posthog/models/person.py

```diff
@@ -13,6 +13,11 @@ def create(self, *args: Any, **kwargs: Any):
             person.add_distinct_ids(distinct_ids)
             return person
 
+    @staticmethod
+    def distinct_ids_exist(team_id: int, distinct_ids: List[str]) -> bool:
+        if PersonDistinctId.objects.filter(team_id=team_id, distinct_id__in=distinct_ids).count() == len(distinct_ids):
```

Comment:
Won't `len(distinct_ids)` always be one? I also think `.exists()` is more efficient than `.count()` but could be wrong

Reply:
It's a many-to-many field so I figured so I figured this was more robust.

`count` should be the same cost as `exists` it pushes it all to the db so marginally more expensive..but not noticeable from my testing.

I can switch this back if we want the simplicity of `.exists`. That was how I had it initially.

Reply:
Yeah let's do exists, this is a bit confusing. Thanks!